cabot/cabotapp/tests/tests_basic.py

```diff
@@ -932,6 +934,16 @@ def test_alert(self, fake_send_alert):
         self.assertEqual(fake_send_alert.call_count, 1)
         fake_send_alert.assert_called_with(self.service, duty_officers=[])
 
+    def test_update_plugins(self):
+        # Test that disabling a plugin is detected by update_alert_plugins
+        plugins = update_alert_plugins()
+        plugin_count = len(plugins)
+        new_apps = [s for s in settings.INSTALLED_APPS if s not in ['cabot_alert_hipchat']]
```

Comment:
just picked a random one. It's just to make sure it updates properly after a plugin is removed